pkg/ui/Makefile

```diff
@@ -75,7 +75,7 @@ $(GOBINDATA_TARGET): webpack.app.js $(DLLS) $(shell find src styl)
 watch: export TARGET := http://localhost:8080
 watch: PORT := 3000
 watch: $(DLLS)
-	$(WEBPACK_DEV_SERVER) --config webpack.app.js --port $(PORT)
+	./node_modules/.bin/webpack-dashboard -- $(WEBPACK_DEV_SERVER) --config webpack.app.js --port $(PORT)
```

Comment:
Can you also wrap webpack-dashboard in $(NODE_RUN), and then stick that in a variable named `WEBPACK_DASHBOARD` at the top of the file? I know we don't currently call `make watch` from CI, but I never ever ever want to have to debug the issue detailed in `node-run` again.

Reply:
Didn't do that intentionally.  Looks like for that to work properly we'd need to address your TODO regarding isatty.  I thought that would be more work that is worth it here.  Also, I think it's highly unlikely that the make watch task would be run within docker.

Reply:
Ah, ok, fair enough. LGTM.